website/src/components/Homepage/Features/Features.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import './Features.css';
 
-const Features = ({ Svg, title, description }) => {
+const Features = ({ Svg, image, title, description }) => {
+  const isSafari = !!navigator.userAgent.match(/Version\/[\d\.]+.*Safari/);
```

Comment:
Please add a comment since that's not a trivial workaround, something like "Since Safari isn't particularly brilliant at rendering SVGs, we use PNG images for the icons."